templates/_renders_job.yaml

```diff
@@ -5,12 +5,46 @@ apiVersion: batch/v1
 
 {{ include "ph.meta.render" . }}
 
+{{ include "ph.job_spec.render" . }}
+
+{{- end -}}
+
+{{- define "ph.job_spec.render" -}}
+
 {{ $spec := include "ph.pod.render" . | fromYaml }}
 
 {{ $_ := set $spec "restartPolicy" "Never" }}
 
 spec:
-  template:
-    spec: {{ $spec | toYaml | nindent 6 }}
+  template: 
+    spec: {{ $spec | toYaml | nindent 8 }}
```

Comment:
Dont really know, but it was 6 before at the same identation.

Reply:
Fixed @AlbertoFemenias , thx!